app/eligibility/process-un-eligible.js

```diff
@@ -0,0 +1,6 @@
+const processUnEligible = (interesRequest) => {
+  console.log('Following interest not Eligible')
+  console.log(interesRequest)
+}
+
+module.exports = processUnEligible

```

Comment:
nitpick - can we called this 
```suggestion
module.exports = processIneligible
```